src/ts/components/seekbar.ts

```diff
@@ -408,7 +408,14 @@ export class SeekBar extends Component<SeekBarConfig> {
     if (this.config.smoothPlaybackPositionUpdateIntervalMs !== SeekBar.SMOOTH_PLAYBACK_POSITION_UPDATE_DISABLED) {
       this.configureSmoothPlaybackPositionUpdater(player, uimanager);
     }
-    this.configureMarkers(player, uimanager);
+
+    // Initialize markers
+    const timelineMarkerConfig = {
```

Comment:
Nit:  would move these 3 lines of code in a separate function by itself instead of adding the additional comment on top

Reply:
In the future-future,  even a refactor of the top bit and moving the top lines into their own separate functions as well